microsoft-edge/dualengine/reference/index.md

```diff
@@ -0,0 +1,35 @@
+---
+description: DualEngine Win32 C++ Reference
+title: DualEngine Win32 C++ Reference
+author: MSEdgeTeam
+ms.author: edgededev
+ms.prod: microsoft-edge
+ms.date: 11/16/2023
+keywords: dual engine, dualengine, iemode, win32 apps, win32, edge, ie mode
+---
+# Reference (DualEngine Win32 C++)

```

Comment:
```suggestion
# DualEngine API Reference
```
alts:
* DualEngine API Reference
* API Reference for DualEngine Win32 C++
* Reference (DualEngine Win32 C++)

rendered:
https://review.learn.microsoft.com/en-us/microsoft-edge/dualengine/reference/?branch=pr-en-us-2859

The TOC node text in Edge Dev docs 
"DualEngine API Reference"
near https://review.learn.microsoft.com/en-us/microsoft-edge/dualengine/get-started?branch=pr-en-us-2859
is different than this top-of-page title

always consider a title or heading along with the first sentence.  useful pattern: short title | short heading, + verbose first sentence to add qualifiers that would be distracting in title|heading.

Reply:
# DualEngine Win32 C++ Reference